positron/comms/ui-frontend-openrpc.json

```diff
@@ -216,6 +216,26 @@
 				}
 			]
 		},
+		{
+			"name": "execute_command_await",
+			"summary": "Execute a Positron command and wait for it to return",
+			"description": "Use this to execute a Positron command from the backend (like from a runtime) and wait for it to return",
+			"params": [
+				{
+					"name": "command",
+					"description": "The command to execute",
+					"schema": {
+						"type": "string"
+					}
+				}
+			],
+			"result": {
+				"schema": {
+					"type": "boolean",
```

Comment:
I think this should return an any type so that command results can be plumbed back to R/Python. That might require changes to the generator, though, so not essential to land this PR.